Technical Charter.md

```diff
@@ -1,242 +0,0 @@
-
-    2.  The scope of the Project includes collaborative development under the
-        Project License (as defined herein) supporting the mission, including
-        documentation, testing, integration and the creation of other artifacts
-        that aid the development, deployment, operation or adoption of the open
-        source project.
-
-2.  Technical Steering Committee
-
-    1.  The Technical Steering Committee (the “TSC”) will be responsible for all
-        technical oversight of the open source Project.
-
-    2.  The TSC voting members are initially the Project’s Committers. At the
-        inception of the project, the Committers of the Project will be as set
-        forth within the “CONTRIBUTING” file within the Project’s code
-        repository. The TSC may choose an alternative approach for determining
-        the voting members of the TSC, and any such alternative approach will be
-        documented in the CONTRIBUTING file. Any meetings of the Technical
-        Steering Committee are intended to be open to the public, and can be
-        conducted electronically, via teleconference, or in person.
-
-    3.  TSC projects generally will involve Contributors and Committers. The TSC
-        may adopt or modify roles so long as the roles are documented in the
-        CONTRIBUTING file. Unless otherwise documented:
-
-        1.  Contributors include anyone in the technical community that
-            contributes code, documentation, or other technical artifacts to the
-            Project;
-
-        2.  Committers are Contributors who have earned the ability to modify
-            (“commit”) source code, documentation or other technical artifacts
-            in a project’s repository; and
-
-        3.  A Contributor may become a Committer by a majority approval of the
-            existing Committers. A Committer may be removed by a majority
-            approval of the other existing Committers.
-
-    4.  Participation in the Project through becoming a Contributor and
-        Committer is open to anyone so long as they abide by the terms of this
-        Charter.
-
-    5.  The TSC may (1) establish work flow procedures for the submission,
-        approval, and closure/archiving of projects, (2) set requirements for
-        the promotion of Contributors to Committer status, as applicable, and
-        (3) amend, adjust, refine and/or eliminate the roles of Contributors,
-        and Committers, and create new roles, and publicly document any TSC
-        roles, as it sees fit.
-
-    6.  The TSC may elect a TSC Chair, who will preside over meetings of the TSC
-        and will serve until their resignation or replacement by the TSC. **The
-        TSC Chair, or any other TSC member so designated by the TSC, will serve
-        as the primary communication contact between the Project and the Urban Computing Foundation of The Linux Foundation.**

```

Comment:
I didn't carry over this TSC Chair language - but if have a need for something like this, we can bring it over. All this says is the chair is the "primary communicator with the UFC and Linux Foundation".. I don't think we need that for OpenJS. Robin, the CPC, etc. should be expect to be able to communicate with any TSC member in opinion.

I think @Pessimistress is the last chair we elected. Wdyt?

Reply:
What do you have in mind?

Reply:
Join the CPC as regular members: https://github.com/openjs-foundation/cross-project-council?tab=readme-ov-file#regular-members